distributed/minGPT-ddp/mingpt/main.py

```diff
@@ -20,7 +24,7 @@ def get_train_objs(gpt_cfg: GPTConfig, opt_cfg: OptimizerConfig, data_cfg: DataC
 
     gpt_cfg.vocab_size = dataset.vocab_size
     gpt_cfg.block_size = dataset.block_size
-    model = GPT(gpt_cfg)
+    model = GPT(gpt_cfg).to(torch.accelerator.current_accelerator())
```

Comment:
try with
```suggestion
    model = GPT(gpt_cfg).to(rank)
```

you may need to add `rank` as an argument of the function